### PR TITLE
Salt master minion test

### PR DESCRIPTION
**Pull request by dgutu-suse**
I've change the code in the way in which salt-master gets "True" when contacting salt-minion.

### CODE REVIEW

**Comment by dgutu-suse** [comment]
Working example - http://markeb.arch.suse.de/tests/761

**Comment by dgutu-suse** [comment]
The error messages in terminal from salt-master were already reported:
https://github.com/saltstack/salt/issues/40912
Theoretically, the next version of salt which will be available in the ASMM repo should contain the fix.

File: tests/console/salt.pm [hunk 1]
```diff
@@ -15,20 +15,24 @@
...
 systemctl start salt-minion
-yes | salt-key --accept-all ; [ "$?" = "141" ] || [ "$?" = "0" ]
-salt '*' test.ping
-systemctl stop salt-master salt-minion
+systemctl status salt-minion
+salt-key --accept-all -y
 EOF
     assert_script_run($_) foreach (split /\n/, $cmd);
+    validate_script_output $output, sub { m/True/ };
+    script_run 'systemctl stop salt-master salt-minion';
```
Comment (okurz) [inline tests/console/salt.pm]: Why should we ignore the exit code from stopping?

File: tests/console/salt.pm [hunk 2]
```diff
@@ -15,20 +15,24 @@
...
 sed -i -e "s/#master: salt/master: localhost/" /etc/salt/minion
 systemctl start salt-minion
-yes | salt-key --accept-all ; [ "$?" = "141" ] || [ "$?" = "0" ]
-salt '*' test.ping
-systemctl stop salt-master salt-minion
+systemctl status salt-minion
+salt-key --accept-all -y
 EOF
     assert_script_run($_) foreach (split /\n/, $cmd);
+    validate_script_output $output, sub { m/True/ };
```
Comment (okurz) [inline tests/console/salt.pm]: Isn't that the same as just calling `salt '*' test.ping | grep -woh True` ? The grep will exit successfully if there *is* true so why so complicated?

**Pull request merged**